cmd/laas/main.go

```diff
@@ -65,9 +65,10 @@ func main() {
 	api.DB = database
 
 	r := gin.Default()
-
+	r.NoRoute(api.HandleInvalidUrl)
 	r.GET("/api/licenses", api.GetAllLicense)
 	r.GET("/api/license/:shortname", api.GetLicense)
-
+	r.POST("/api/license", api.CreateLicense)
+	r.PATCH("/api/license/update/:shortname", api.UpdateLicense)
```

Comment:
Please move them into api package itself, and factor out functions so that they can be reused in tests. Do the same with all the database steps you are doing. Move them all into a db module and make necessary changes.